### DISCUSSION TITLE
Blank area at the top of the meeting list on all tested devices and browsers

### DISCUSSION THREAD

**Discussion by Servant1212** [code4recovery/12-step-meeting-list#1411] [General]
Blank area at the top of the meeting list on all tested devices and browsers
div#meetings.container 1260X550 appears when inspecting the element.  this element is <div id="meetings" their is no height key in the div tag nor inline css .  This tag is within the <div id="tsml"> </div>
![Capture](https://github.com/code4recovery/12-step-meeting-list/assets/148838193/cd738802-92a3-4485-852b-f04ea38793e0)

**Comment by thessnake03** [comment]
Looks like it's padding all around.  The sites I checked had 16 px on the sides, top and bottom.  I was able to remove it with css, but it looks a bit odd without the side margins.

```
div#meetings.container{
	padding: 0px!important;
}
```

**Comment by Servant1212** [comment]
I think that is a major improvement.  At least people will not leave without scrolling down.
________________________________

**Comment by Servant1212** [comment]
This seemed to work for a time but I was working on widgets recently and noticed that someone had attempted to modify the meeting widget to include a donate link.  Upon working on the widget I needed to work on the issue returned.  I attempted to remove the donate link from the header widget but this has had no impact on the blank area.  Is it possible to reload the widget?
________________________________

**Reply by joshreisner** [reply]
if you could provide a url to the affected page and a screenshot of how you want it to look that could help us understand the problem

sounds like there are widgets above the meeting list that are pushing it down? perhaps removing / deleting them would fix the issue?
